src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -154,6 +233,14 @@ public CommonTermsQueryBuilder highFreqMinimumShouldMatch(String highFreqMinimum
         return this;
     }
 
+    public String highFreqMinimumShouldMatch() {
+        return this.highFreqMinimumShouldMatch;
+    }
+
+    public CommonTermsQueryBuilder minimumShouldMatch(String minimumShouldMatch) {
```

Comment:
how do we set the highFreqMinimumShouldMatch ? maybe I missed it... and why is this setter called differently?